docs/examples/junit4/redis/src/test/java/quickstart/RedisBackedCacheIntTest.java

```diff
@@ -14,15 +15,16 @@ public class RedisBackedCacheIntTest {
 
     // rule {
     @Rule
-    public GenericContainer redis = new GenericContainer(DockerImageName.parse("redis:6-alpine"))
-        .withExposedPorts(6379);
+    public TestcontainersRule<GenericContainer<?>> redis = new TestcontainersRule<>(
```

Comment:
I suggest naming this field `redisRule` (or, possibly, `redisContainerRule`).

This test is apparently used in the documentation, and having a more intention-revealing name could reduce confusion.

Reply:
Previously the fields were not named "...rule", and this is a "best effort to keep running with outdated junit4", so I would really prefer to refrain from renaming lots of fields.

Reply:
Preciously they fields had a different type